index.js

```diff
@@ -1,3 +1,11 @@
+const isCompressed = (res) => {
```

Comment:
It actually force people to use `.toString('base64')` when `'content-encoding' is exist and not 'identity'` and losing the ability of customization.
I recommend allow overwrite and put it by default.

Which means I want the following pattern.
```js
const isCompressedDefault = (res) => {
  const contentEncoding = res.headers['content-encoding'] || res.headers['Content-Encoding']
  return contentEncoding && contentEncoding !== 'identity'
}

const customBinaryCheck = (options, res) => {
  const enforceBase64 = typeof options.enforceBase64 === 'function' ? options.enforceBase64 : isCompressedDefault 
  return enforceBase64(res) === true
}
```

Reply:
That's reasonable. Changed to follow this pattern.